megatron/mup_substitute.py

```diff
@@ -39,31 +41,51 @@ def _get_coord_data(
 ):
```

Comment:
To implement the coord check heres what I did:

`megatron/training.py::train()`
![image](https://github.com/EleutherAI/gpt-neox/assets/10405248/4684e77b-03a4-479a-9184-e5176b6d9021)

`megatron/training.py::training_log()`
![image](https://github.com/EleutherAI/gpt-neox/assets/10405248/f9db8975-6a16-4a07-9d94-05cad2dbad30)

Then to run a coord check I just configure a 10 step training run where these scalars are being logged. To make the coord check graphs you can just parse your tensorboard logs 
cc @lintangsutawika @Quentin-Anthony

Reply:
Then in `arguments.py` I add this to enable coord check:
![image](https://github.com/EleutherAI/gpt-neox/assets/10405248/6b71b2aa-13b9-4a0d-bc4b-db8f4992916a)